### ISSUE TITLE
[feature request] Add proteins and e.g. fat on db

### ISSUE THREAD

**Issue by maste9** [davidhealey/waistline#14]
For mere weight regulation the app does  what it should. For a bit more sophisticated diet surveillance further nutrition markers would help. E.g. for me as long distance runner proteins would be a nice to have information about my daily consumption of them.

**Comment by davidhealey** [comment]
Yes this has already been requested but since it's not something that I have use for I don't plan to add it to the current app. However I am working on a native android rewrite which will include this feature but probably won't be available for a few months.

**Comment by davidhealey** [comment]
Cordova is a bit buggy and doesn't work well on old devices. I originally intended to make a native app but then figured Cordova would be easier (which it is) but I would still prefer a native app and also the app seems to be quite popular and to include some of the suggested features would be easier in a native app and it would be more efficient.
